model-optimizer/mo/graph/graph.py

```diff
@@ -147,7 +147,10 @@ def in_ports(self, control_flow=False):
         for idx in self._in_ports:
             if control_flow or 'control_flow' not in self._in_ports[idx] or not self._in_ports[idx]['control_flow']:
                 ports.update({idx: self.in_port(idx, control_flow=control_flow)})
-        return dict_to_ordered_dict(ports, func=lambda t: str(t))
+        if control_flow:
+            return dict_to_ordered_dict(ports, func=lambda t: str(t))
```

Comment:
so if we have a node with more than 9 control flow inputs then we will have the original error. Looks like a partial solution :)

Reply:
Also, I would like to see unit tests covering this very low-level code

Reply:
Done, I did refactor for sorting and add tests.